iphone/Classes/TiCalendarAttendee.m

```diff
@@ -0,0 +1,98 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_CALENDAR
+
+#import "CalendarModule.h"
+#import "TiCalendarAttendee.h"
+
+@implementation TiCalendarAttendee
+
+#pragma mark - Internals
+
+-(id)_initWithPageContext:(id<TiEvaluator>)context participant:(EKParticipant*)participant_ isOrganiser:(BOOL)_isOrganiser module:(CalendarModule*)module_
+{
+	if (self = [super _initWithPageContext:context]) {
+		module = [module_ retain];
+		participant = [participant_ retain];
+		isOrganiser = _isOrganiser;
+	}
+	return self;
+}
+
+-(void)dealloc
+{
+	RELEASE_TO_NIL(module);
+	RELEASE_TO_NIL(participant);
+	[super dealloc];
+}
+
+-(EKParticipant*) participant
+{
+	return participant;
+}
+
+-(NSString*)apiName
+{
+	return @"Ti.Calendar.Attendee";
+}
+
+
+#pragma mark - Public API's
+
+-(id)valueForUndefinedKey:(NSString *)key
```

Comment:
So why are we not creating native getter instead of overriding this system-method? E.g.:

```
-(NSString*)name
{
    return [[self participant] name];
}
```

Also note that we should have getter to create the instance variables `participant` and `module`.